go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/stretchr/testify v1.4.0
 	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
+	golang.org/x/text v0.3.2
```

Comment:
I'm not seeing why this got pulled in, and `go mod tidy` removes it.

Reply:
I am not sure too, removed it.